src/components/DNSMap/DNSMap.tsx

```diff
@@ -110,15 +110,15 @@ const DNSMap = () => {
         const args = [`${values.domain}`, "-d", `${values.delay}`];
 
         if (values.wordlistPath) {
-            args.push(`-S ${values.wordlistPath}`);
+            args.push(`-w`, `${values.wordlistPath}`);
         }
 
         if (values.csvResultsFile) {
-            args.push(`-s ${values.csvResultsFile}`);
+            args.push(`-c`, `${values.csvResultsFile}`);
```

Comment:
### Suggestion for improvment
 For now the if statement is completely fine but if you want to improve the scalability of the code for future work you could look into expression such as ternary operator or a a better way of doing if statement.